src/common-js/misc.js

```diff
@@ -1 +1,28 @@
 export const isUndefined = (v) => typeof v === 'undefined'
+
+const snakeStrToCamel = (str) =>
+  str.replace(/([-_][a-z])/g, (group) => group.toUpperCase().replace('-', '').replace('_', ''))
+
+export const snakeToCamel = (obj) => {
+  if (typeof obj != 'object') return obj
```

Comment:
so, if you do `typeof null`, you also get `'object'`.

I believe this is a better implementation: https://matthiashager.com/converting-snake-case-to-camel-case-object-keys-with-javascript

Besides, I think the method should be called something like `objectKeysToCamelCase`.

Reply:
good catch let's add a test for that

Reply:
so, i just added a test and the current implementation works with null values.

Reply:
i also like the implementation you suggested better. It returns a new object instead of  mutation the original object in place. Plus it is more concise. Will change it.